packages/nc-gui/components/smartsheet/header/CellIcon.vue

```diff
@@ -85,5 +85,5 @@ const icon = computed(() => {
 </script>
 
 <template>
-  <component :is="icon" class="text-grey mx-1 !text-sm" />
+  <component :is="icon" class="text-grey mx-1 !text-xs" />
```

Comment:
Should the icons really be reduced in size? They're already pretty small.

Reply:
Navi wants it smaller